NitroxPatcher/Patches/Dynamic/GoalManager_OnCompletedGoal_Patch.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Core;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
```

Comment:
Minor: In general we remove unused usings.

Reply:
Removed in commit #4.